src/App.test.js

```diff
@@ -1,9 +1,57 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import assert from 'assert';
+import * as PIXI from 'pixi.js';
+
+import settings from './config';
+import Communication from './components/Communication';
 import App from './App';
+import GamemodeHandler from './game/GamemodeHandler';
+import Game from './game/Game';
 
 it('renders without crashing', () => {
   const div = document.createElement('div');
   ReactDOM.render(<App />, div);
   ReactDOM.unmountComponentAtNode(div);
 });
+
+describe('GamemodeHandler', () => {
+  it('is a working singleton', () => {
+    const inst1 = GamemodeHandler.getInstance();
+    const inst2 = GamemodeHandler.getInstance();
+
+    assert.equal(inst1, inst2);
```

Comment:
Suggest using Jests built ins instead of external 'assert' library. https://facebook.github.io/jest/docs/en/expect.html